loop.go

```diff
@@ -12,7 +13,9 @@ import (
 )
 
 const (
-	retrySectionKey = "retry"
+	loopSectionKey            = "loop"
+	deprecatedRetrySectionKey = "retry" // deprecated
+	loopCountVarKey           = "i"
```

Comment:
[nits]
How to name the key if we try to implement a counter for the index of steps[*] in a future feature addition? I was wondering.

Is there a way to concisely describe a case that references an index when testing a response in the same step?

```yml
steps:
  -
    req:
      /get?var=foo:
        get:
          body: null
    test: |
      steps[0].res.status == 200
```

I wanted to be concise, e.g., to be able to refer to the index of `steps[0]` by a variable.

Reply:
I have the same problem as you (To solve this problem, I created a step listing with a [map](https://github.com/k1LoW/runn#runbook--runn-scenario-file-).

> How to name the key if we try to implement a counter for the index of steps[*] in a future feature addition? I was wondering.

I'm certainly wondering what to do about it.

If we were to support both Array and Map types, should there be a way to specify the current step as follows?

```yaml
steps:
  -
    req:
      /get?var=foo:
        get:
          body: null
    test: |
      current.res.status == 200
```

```yaml
steps:
  reqget:
    req:
      /get?var=foo:
        get:
          body: null
    test: |
      current.res.status == 200
```

Reply:
If you have any ideas for naming the step[*] counter and the loop counter well, please let me know!

Reply:
> If we were to support both Array and Map types, should there be a way to specify the current step as follows?

The `current` key looks very nice, since it can be used in either Map or Array.

I'd like to respond separately from this PR.

Reply:
@k2tzumi Thank you very much!!

Reply:
@k2tzumi I invited you to become a runn collaborator!